docker/python3.6_ci/src/run.sh

```diff
@@ -27,6 +35,7 @@ function run_tests {
   echo "Testing ..."
 
   install_dependencies
+  FIND_MATCH_PATHS=${FIND_MATCH_PATHS:=/data}
```

Comment:
See here: https://stackoverflow.com/a/27445490/1558022

We want to assign this even if the variable is unset, not just set and null.